examples/chip-tool/commands/clusters/ClusterCommand.h

```diff
@@ -141,6 +124,23 @@ class ClusterCommand : public InteractionModelCommands, public ModelCommand, pub
         }
     }
 
+protected:
+    ClusterCommand(const char * commandName, CredentialIssuerCommands * credsIssuerConfig) :
+        InteractionModelCommands(this), ModelCommand(commandName, credsIssuerConfig)
+    {
+        // Subclasses are responsible for calling AddArguments.
+    }
+
+    void AddArguments()
```

Comment:
Is it worth renaming this to something like AddCommonArguments() to indicate that we're adding a common set of arguments?

Reply:
The problem is that subclasses will call it as AddArguments....  We could have AddCommonArguments that we call from both AddArguments and the constructors, I guess, but we need an AddArguments implementation.